scripts/launch/connect-to-cluster/mainnet-exchange.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+readonly CLUSTER=mainnet
+
+flush_wallet=""
+
+if [[ "$1" == "-c" ]]; then
+  shift
+  flush_wallet="--flush-wallet-db"
+fi
+
+echo "Launch a single node and connect it to '${CLUSTER}' cluster..."
+
+readonly TMP_TOPOLOGY_YAML=/tmp/topology.yaml
+printf "wallet:
+    relays: [[{ host: relays.cardano-mainnet.iohk.io }]]
```

Comment:
If we're doing this, we should use the topology exchanges are actually using.

```
prinf 'wallet:
 relays: [ [ {"host": "cardano-node-0.cardano-mainnet.iohk.io"}
           , {"host": "cardano-node-1.cardano-mainnet.iohk.io"}
           , {"host": "cardano-node-6.cardano-mainnet.iohk.io"}
           ]
         , [ {"host": "cardano-node-2.cardano-mainnet.iohk.io"}
           , {"host": "cardano-node-3.cardano-mainnet.iohk.io"}
           , {"host": "cardano-node-6.cardano-mainnet.iohk.io"}
           ]
         , [ {"host": "cardano-node-4.cardano-mainnet.iohk.io"}
           , {"host": "cardano-node-5.cardano-mainnet.iohk.io"}
           , {"host": "cardano-node-6.cardano-mainnet.iohk.io"}
           ]
         ]' >i"${TMP_TOPOLOGY_YAML}"
```

Reply:
Makes total sense, thanks!